pkg/remote/client.go

```diff
@@ -0,0 +1,144 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api/write"
+	"github.com/tendermint/tendermint/libs/log"
+)
+
+const (
+	NodeIDTag  = "node_id"
+	ChainIDTag = "chain_id"
+)
+
+// EventCollectorConfig is the influxdb client configuration used for
+// collecting events.
+type EventCollectorConfig struct {
+	// URL is the influxdb url.
+	URL string `mapstructure:"url"`
+	// Token is the influxdb token.
+	Token string `mapstructure:"token"`
+	// Org is the influxdb organization.
+	Org string `mapstructure:"org"`
+	// Bucket is the influxdb bucket.
+	Bucket string `mapstructure:"bucket"`
+	// BatchSize is the number of points to write in a single batch.
+	BatchSize int `mapstructure:"batch_size"`
+}
+
+// DefaultEventCollectorConfig returns the default configuration.
+func DefaultEventCollectorConfig() *EventCollectorConfig {
+	return &EventCollectorConfig{
+		URL:       "",
+		Org:       "core/app",
```

Comment:
not `"celestia"`?

Reply:
yeah good point, we can switch to that. If for some reason other teams want to use the same influxdb instance, then we might want to switch back, but until then I agree and think celestia makes more sense

https://github.com/celestiaorg/celestia-core/pull/970/commits/014b9dba582c5526e5f1b86f3f2045abd847eba2